src/main/java/jp/co/soramitsu/sora/infrastructure/Platform.java

```diff
@@ -1,15 +1,12 @@
-package infrastructure;
+package jp.co.soramitsu.sora.infrastructure;
```

Comment:
Are you sure about package naming?
Maybe jp.co.soramitsu.sora.qa.infrastructure will be more stable here?
Or Maybe jp.co.soramitsu.sora-qa.infrastructure ?

Reply:
changed to jp.co.soramitsu.sora.qa